video_xblock/video_xblock.py

```diff
@@ -777,18 +785,18 @@ def upload_manual_transcript_handler(self, request, suffix=''):
         course_key = CourseKey.from_string(course_key_string)
         filename = upload_file.name.replace(" ", "_")
 
-        # Files in the .vtt format are saved without conversion.
+        # All upload files (except for .srt) are saved without conversion.
         # The transcript in the .srt format should be converted to the .vtt format
         # and then saved using the standard flow.
-        if not filename.endswith('.vtt'):
+        if filename.endswith('.srt'):
```

Comment:
I would store the `'.srt'` value in the constant of type tuple, e.g. `SUPPORTED_NON_VTT_TRANSCRIPTS`.

It would ease the supported transcripts list extension in the future.

But we can leave it as it is for now.

Reply:
all the same, I decided to leave everything as it is for now since it is not clear how this X-block will develop further

Reply:
ok